docs/dev/src/antora.yml

```diff
@@ -3,3 +3,4 @@ version: master
 
 nav:
 - modules/java/nav.adoc
+- modules/spring/nav.adoc
```

Comment:
When we build it locally, we also want to access the spring sections. That may change later when we integrate with the main docs. But we can go with this approach for now.

Reply:
Won't this mean the spring docs will be visible if we merge this and then release the sdk?

Reply:
hmm, you are right

Reply:
It's now removed